apps/web/modules/ui/components/id-badge/index.test.tsx

```
@@ -0,0 +1,139 @@
+  test("applies custom className", () => {
+    const { container } = render(<IdBadge id="STYLE-TEST" className="custom-class" />);
+
+    const badge = container.querySelector(".custom-class");
+    expect(badge).toBeInTheDocument();
+    expect(badge).toHaveClass("custom-class");
+  });
+
+  test("disables copy button when disabled prop is true", () => {
+    const { container } = render(<IdBadge id="DISABLED-TEST" />);
+
+    const copyButton = container.querySelector('button[aria-label="Copy DISABLED-TEST"]')!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)